lesson3.slide

```diff
@@ -283,8 +283,8 @@ Outline:
 - *dynamic* *type*: type of assigned _value_
 - *dynamic* *value*: the actual value assigned
 - where's the problem?
-- interface type value is `nil` iff both dynamic value *and* dynamic type are `nil`
-- sometimes we get the impression that `nil != nil`
+- interface type value is `nil` if both dynamic value *and* dynamic type are `nil`
+- sometimes we get the impression that `nil`!=`nil`
```

Comment:
looks ok, thank you very much!

@dmvolod are you taking this course?

Reply:
Yes, @tisnik , very good content. Thank you.

Reply:
thank you. Good to know!